src/Enum/Edit.ts

```diff
@@ -0,0 +1,6 @@
+export enum Edit {
```

Comment:
```suggestion
export enum Action {
```
Sounds better in this case